xdsl/parser/core.py

```diff
@@ -764,7 +777,7 @@ def _parse_op_result(self) -> tuple[Span, int]:
     def _parse_op_result_list(self) -> list[tuple[Span, int]]:
         """
         Parse the list of operation results.
-        If no results are present, returns an empty list.
+        If no results are present, return an empty list.
```

Comment:
Can you drop this change, as this is unrelated.

Reply:
Sorry for introducing unrelated changes.